src/components/Home.jsx

```diff
@@ -39,14 +69,21 @@ const Home = ({ setActors, actors, currentScript, setCurrentScript }) => {
         <Upload setCurrentScript={setCurrentScript} setTitles={setTitles} titles={titles} />
         <form onSubmit={newActorHandler}>
           <input ref={newActorInput} placeholder="enter actor's name" />
+          <div style={{ color: 'red' }}>{addError}</div>
           <button type="submit">Add actor</button>
         </form>
       </div>
       <div className="row">
         <ul id="actorList">
           <h3>Actors</h3>
+          <div style={{ color: 'red' }}>{deleteError}</div>
           {actors.map((actor) => (
-            <li key={actor.id}>{actor.name}</li>
+            <li key={actor.id}>
+              <button onClick={deleteActor} className="delete" value={`${actor.name},${actor.id}`}>
+                -
```

Comment:
In the future UI, it would be nice to have a trash can icon with the text "delete" so it's both recognizable and accessible.

Reply:
Additionally, would be nice to have some sort of protection here, such as one of these:
* A modal to confirm deletion
* An undo button if someone accidentally deletes an actor
* Have delete buttons hidden by default and require pressing a "allow deletions" button before they show up as a sort of lock mode to prevent accidental clicks on the delete buttons.